README.md

```diff
@@ -1,4 +1,34 @@
 # Template for webots.cloud simulations
-Use this repository as a template to deploy a simulation on webots.cloud.
+Use this repository as a template to deploy a simulation on [webots.cloud](webots.cloud).
+
+## Structure
+The template contains a generic project featuring the e-puck robot.
+The world `e-puck.wbt` is located in the `worlds` folder.
+The controller `e-puck.c` is located in the `controllers/e-puck` folder.
+The `remote_controls` plugin, as well as the `e-puck` robot window, are located in the `plugins` folder.
+The `E-puck.proto` PROTO is located in the `protos` folder.
+
+The defined `Dockerfile` is placed at the root of the project and implements the four lines of `Dockerfile.default`, which would have been used by [webots.cloud](webots.cloud) if no Dockerfile was defined.
+
+```dockerfile
+FROM cyberbotics/webots:R2022b-ubuntu20.04
+ARG PROJECT_PATH
+RUN mkdir -p $PROJECT_PATH
+COPY . $PROJECT_PATH
+```
+
+The following two additional lines allow to compile the controller and the plugin when [webots.cloud](webots.cloud) creates the Docker image.
+
+```dockerfile
+RUN cd $PROJECT_PATH/controllers/e-puck && make clean && make
+RUN cd $PROJECT_PATH/plugins/remote_controls && make clean && make
+```
```

Comment:
I think you also need to compile the robot window.

I don't know if it make a lot of sense to put the remote_controls plugin as the user will not be able to test it in most of the case